docker-app/qfieldcloud/core/serializers.py

```diff
@@ -523,3 +523,43 @@ class LatestPackageSerializer(serializers.Serializer):
     package_id = serializers.UUIDField()
     packaged_at = serializers.DateTimeField()
     data_last_updated_at = serializers.DateTimeField()
+
+
+
+# Add the necessary imports for Team and TeamMember models
+from qfieldcloud.core.models import Team, TeamMember
+
+
+class TeamSerializer(serializers.ModelSerializer):
+    """
+    Serializer for creating a Team, accepts only 'username', 'team_organization', and optional 'members'.
+    """
+    
+    # members = serializers.PrimaryKeyRelatedField(queryset=OrganizationMember.objects.all(), many=True, required=False)
```

Comment:
Do we need the `members` variable? If not, please remove commented code, nobody is going too look for it in the future.
 If you really need it, please add a `# TODO we commented this because X and Y` on the line before.

Reply:
Has been addressed!